packages/website/lib/magic.js

```diff
@@ -1,20 +1,21 @@
-import { Magic } from 'magic-sdk'
-import { OAuthExtension } from '@magic-ext/oauth'
-import constants from './constants'
+import { Magic } from 'magic-sdk';
```

Comment:
so many semis

Reply:
im removin them and will later take on https://github.com/web3-storage/web3.storage/issues/1762